_posts/2013-08-07-employees.markdown

```diff
@@ -16,6 +16,7 @@ title: Employees
 | `middle_initial`              | String            |           |    X     | null    | employee's middle initial. Not guaranteed to exist.
 | `last_name`                   | String            |           |          |         | employee's last name.
 | `email`                       | String            |           |    X     | null    | employee's email address. This is provided specifically to sync users between our system and yours. You may not use this address for any other purpose (e.g. marketing.) Not guaranteed to exist.
+| `department`                  | String            |     X     |          | null    | employee's department.
```

Comment:
Not sure if I get the right doc, because there are other fields that don't match the rabl file

Reply:
There are also company_id and manager_id

Reply:
Ah, yes. Can you please add those? Thanks!